packages/appium/docs/en/developing/build-plugins.md

```diff
@@ -412,6 +412,12 @@ Overriding Execute Methods from drivers works as you'd expect: if your plugin de
 Method with the same name as that of a driver, your command (in this case `plugMeIn`) will be
 called first. You can choose to run the driver's original behaviour via `next` if you want.
 
+### Build Appium Doctor checks
+
+Your users can run `appium driver doctor <pluginName>` to run installation and health checks. Visit
```

Comment:
This should probably be `appium plugin doctor <pluginName>` :)

Reply:
ah right you are!